package.json

```diff
@@ -45,13 +45,14 @@
     "eth-store": "^1.1.0",
     "ethereumjs-tx": "^1.0.0",
     "ethereumjs-util": "^4.4.0",
+    "gulp": "github:gulpjs/gulp#4.0",
```

Comment:
We no longer require this weird version of gulp, as long as you're using node 6+, you should be able to use the main version on npm.